src/tests/daos_perf.c

```diff
@@ -1350,8 +1359,8 @@ main(int argc, char **argv)
 				return rc;
 		}
 	} else {
-		if (ts_in_ult) {
-			fprintf(stderr, "ult is only supported for VOS mode\n");
+		if (ts_in_ult || ts_profile_vos) {
+			fprintf(stderr, "ult/prof only supported VOS mode\n");
```

Comment:
I think it would be OK to say "ult/prof is only supported in VOS mode"  Alternatively, make the string specific to the variable that was true.  It's not super important and I would not hold up approval of it, but if you are changing the string anyway, I think its worth looking at making the change.

Reply:
oh, this is only for satisfying the 80 line limit.  but I will update the error message here. Thanks.